sema/subyt/sources.py

```
@@ -3,7 +3,7 @@
 import mimetypes
 import os
 from pathlib import Path
-from typing import Callable
+from typing import Callable, Iterable
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)